### PR TITLE
Relax a couple of logging instructions to reduce unnecessary noise

### PR DESCRIPTION
**Pull request by rodnymolina**
```
time="2021-07-07 19:31:49" level=warning msg="Failed to wait for seccomp session: Interrupted poll operation: closed file-descriptor"
time="2021-07-07 19:33:08" level=warning msg="Failed to wait for seccomp session: Interrupted poll operation: closed file-descriptor"
time="2021-07-07 19:34:12" level=warning msg="Failed to wait for seccomp session: Interrupted poll operation: closed file-descriptor"
time="2021-07-07 19:57:47" level=warning msg="Failed to wait for seccomp session: Interrupted poll operation: closed file-descriptor"
```

```
time="2021-07-04 17:30:32" level=warning msg="TOCTOU check failed on fd 74 pid 31949 cntr 21842f8de4fa: req.Id is no longer valid (no such file or directory)"
```

Signed-off-by: Rodny Molina <rmolina@nestybox.com>

### CODE REVIEW

File: seccomp/tracer.go [hunk 1]
```diff
@@ -372,7 +372,7 @@ func (t *syscallTracer) connHandler(c *net.UnixConn) error {
 		// Return here to exit this goroutine in case of error as that implies
 		// that the seccomp-fd is not valid anymore.
 		if err := t.seccompSessionRead(seccompSession); err != nil {
-			logrus.Warnf("Failed to wait for seccomp session: %v", err)
+			logrus.Debugf("Failed to wait for seccomp session: %v", err)
```
Comment (ctalledo) [inline seccomp/tracer.go:375]: Why this change ... isn't this a serious condition?

**Review by ctalledo — APPROVED** [review]
One minor question ...

File: seccomp/tracer.go (hunk 1 above)
Comment (rodnymolina) [reply seccomp/tracer.go:375]: Not really. This simply indicates that the seccomp-fd isn't available (at least for polling purposes) anymore, which happens coz the sys-container died, or the seccomp-fd simply ceased to exist after user exited a `docker exec` session.
